lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -200,9 +200,10 @@ fn do_test_simple_monitor_temporary_update_fail(disconnect: bool) {
 	let events_3 = nodes[1].node.get_and_clear_pending_events();
 	assert_eq!(events_3.len(), 1);
 	match events_3[0] {
-		Event::PaymentReceived { ref payment_hash, ref purpose, amount_msat } => {
+		Event::PaymentReceived { ref payment_hash, ref purpose, amount_msat, via_channel_id, .. } => {
```

Comment:
Personal preference: I like enumerating all fields + leaving some as unused rather than using `..` where it makes sense, to force future authors to consider whether to assert on new fields they add

Reply:
Done with https://github.com/lightningdevkit/rust-lightning/pull/1856/commits/f7149200c54bcdc7c75ff441fe8ad038140fb29a.